### ISSUE TITLE
Weird outline outside (some) programs 

### ISSUE THREAD

**Issue by philipt4** [vinceliuice/Orchis-kde#13]
I am running Fedora KDE and immediately after installing the Orchis theme and applying it I get this bug.

Opening a program will leave it with a black shadowy outline with a pretty thick gap between the edges of the program and the weird outline. Moving or scaling the program will immediately cause the strange outline to disappear, making everything normal until close then open the program again. 

![weird-discover](https://user-images.githubusercontent.com/35475073/141671347-9a1006d0-6d89-46c6-a3ca-427700a4ca95.png)
![weird-dolphin](https://user-images.githubusercontent.com/35475073/141671348-d6e9601f-5ca7-42a5-b98f-3e573c9f73b3.png)
![weird-settings](https://user-images.githubusercontent.com/35475073/141671349-8378a5bf-7a7f-49db-b0cf-eb4c2b91766c.png)
Is there a fix?

**Comment by vinceliuice** [comment]
Only this theme have this issue?

**Comment by philipt4** [comment]
> Only this theme have this issue?

Yeah, when i switch Window Decoration to any of the four Orchis options. It seems to be only system programs like file browser, discover shop, setitngs stuff like that. Gwenview also. Firefox always opens fine as well as a few other programs.

**Comment by philipt4** [comment]
I should mention I am using Wayland not Xorg. Fedora KDE

**Comment by philipt4** [comment]
> Only this theme have this issue?

Is there a fix for the theme or something i should try?

**Comment by vinceliuice** [comment]
I do not know how to fix this

**Comment by sylviiu** [comment]
+1, same issue here. It feels like this happens to anything native to QT?

Also noting that I'm also running Wayland. I'm going to switch to X11 for a bit to see if it resolves the issue?

![image](https://user-images.githubusercontent.com/95180094/154520729-4eb2b3d4-e0db-4d0d-b24b-6437c24c58d7.png)
